src/Http/routes.php

```diff
@@ -5,7 +5,10 @@
     // See resource routes in Laravel Docs
     Route::resource('Patient', '\LibreEHR\FHIR\Http\Controllers\PatientController');
     Route::resource('Patients', '\LibreEHR\FHIR\Http\Controllers\PatientController');
+    Route::get('Patient/{id}', '\LibreEHR\FHIR\Http\Controllers\PatientController@show($id)');
```

Comment:
@Leo24 These additional resource routes are not needed, they are all covered by the ::resource() route.